common/translation_utils/make_pos.py

```diff
@@ -0,0 +1,136 @@
+import polib
+import csv
+import copy
+import os
+
+class Translation(object):
+    pass
+
+def make_pos_run():
+
+    def make_translation():
```

Comment:
It's pretty uncommon to define functions within other functions.
You can just keep them outside of `make_pos_run`.

Reply:
Have removed nested functions